src/Illuminate/Support/Collection.php

```diff
@@ -1210,7 +1210,7 @@ public function zip($items)
             return new static(func_get_args());
         }, $this->items], $arrayableItems);
 
-        return new static(call_user_func_array('array_map', $params));
+        return array_map(...$params);
```

Comment:
`new static` is missing.

Reply:
This has already been fixed a couple of days ago. You are looking at an outdated diff.

Reply:
You are right. Sorry.